src/components/ProfileSideNav/ProfileSideNav.tsx

```diff
@@ -39,11 +44,13 @@ const ProfileSideNav = () => {
   };
 
   return (
-    <div className="flex flex-col m-2 p-1 border gap-2 w-3/12 h-min">
-      <h1 className="text-lg font-bold">My account</h1>
+    <div
+      className={`${className} flex flex-col p-1 border gap-2 h-min  ProfileSideNav`}
+    >
+      <h1 className="text-lg font-bold m-0 p-2.5">My account</h1>
       <Link
         href="/profile/settings"
-        className={`p-1 ${
+        className={`no-underline block p-2.5 hover:bg-black hover:text-white ${
```

Comment:
You're duplicating a lot of Tailwind classes for all those links here. I would suggest you extract them into a `commonClassNames` variable, and then just use that.